src/pages/home/ReportScreen.js

```diff
@@ -144,7 +140,7 @@ function ReportScreen({
     route,
     report,
     reportMetadata,
-    reportActions,
+    allReportActions,
```

Comment:
`allReportActions` is missing from PropTypes

Reply:
👍🏻

Reply:
Thanks for pointing it up!